person.rb

```diff
@@ -0,0 +1,55 @@
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/227523563-2573e4e0-3c1f-4407-9960-3e2460977ae2.png)

- [ ] It would be great if you fix the Argument error as suggested by the previous reviewer which you can see in the image above :thinking:

### Fix for this issue :arrow_down:

To fix this Argument error kindly pass the default value to parent_permission with an equal sign `(=)` instead of a colon `(:)` as you can see in the image below :blush: :+1:

![image](https://user-images.githubusercontent.com/93039359/227523236-e1dadca6-08f2-45de-9ad1-58502ef88a05.png)

Reply:
![Wrong review](https://user-images.githubusercontent.com/108669691/227552411-9bb1127e-77d9-4aff-8e8a-a10320577928.PNG)


please @HAMMAS-SALEEM  you are wrong, according to rubocop, we are supposed to use (:) and not (=) 

![test parameters](https://user-images.githubusercontent.com/108669691/227553441-250c2a8d-ee0d-4d5f-9902-0f7c1abfabdf.PNG)

 and in the test parameters given by microverse, the boolean true or false was not included.

Reply:
Thanks for reaching out my dear brother @Outkarm
I have checked it and you are right that according to the code snippet. But it raised a point here.

**The person who will check my review**
He/She can tell me about it that if we want to add a student without any permission then how are we gonna pass false in the argument because as you saw in the error image above it won't allow you to pass the other boolean then how are you gonna fix it? I have already given the fix but @Outkarm argued that it is against the linters. Then I would like you to take this issue to your team :+1: